diff --git a/node_modules/@devular/gatsby-plugin-plausible/gatsby-ssr.js b/node_modules/@devular/gatsby-plugin-plausible/gatsby-ssr.js
index da958f6..6974d46 100644
--- a/node_modules/@devular/gatsby-plugin-plausible/gatsby-ssr.js
+++ b/node_modules/@devular/gatsby-plugin-plausible/gatsby-ssr.js
@@ -49,7 +49,7 @@ exports.onRenderBody = function (_ref, pluginOptions) {
     var scriptProps = {
       async: true,
       defer: true,
-      domain: domain,
+      'data-domain': domain,
       src: proxyScript || "https://" + plausibleDomain + scriptURI
     };
 
