diff --git a/node_modules/@lekoarts/gatsby-theme-minimal-blog-core/src/templates/tag-query.tsx b/node_modules/@lekoarts/gatsby-theme-minimal-blog-core/src/templates/tag-query.tsx
index 0c74e61..d13a488 100755
--- a/node_modules/@lekoarts/gatsby-theme-minimal-blog-core/src/templates/tag-query.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-minimal-blog-core/src/templates/tag-query.tsx
@@ -17,6 +17,15 @@ export const query = graphql`
           name
           slug
         }
+        banner {
+          childImageSharp {
+            resize(width: 300, height: 200, quality: 90) {
+              src
+              width
+              height
+            }
+          }
+        }
       }
     }
   }
